pipeline/backend/kubernetes/kubernetes.go

```diff
@@ -120,12 +122,17 @@ func (e *kube) Load(context.Context) error {
 	}
 
 	if err != nil {
-		return err
+		return nil, err
 	}
 
 	e.client = kubeClient
 
-	return nil
+	// TODO: use info resp of kubeClient to define platform var
```

Comment:
Wont be possible as Kubernetes is a cluster and therefore it could have multiple architectures. We can only stick to the architecture of the agents host.

Reply:
Well if it's `linux/amd64`,`linux/riscv64`

We could report `linux/multi` or `linux/{amd64,riscv64}` ...

Reply:
And if we only have one type we do report as with docker?

Reply:
Would be a tough one to match such labels. Maybe adding it as regex could do the trick. Okay, but that's really a future PR I guess

Reply:
I'll create a new issue and link the todo to it ... it is worth it